Trace/Readme.md

```diff
@@ -0,0 +1,11 @@
+# Quickstart
+
+To collect traces:
+- Double-click on Trace.cmd.
+- Run your scenario when told.
+- Press Enter and wait for the script to complete.
+- The folder containing the traces will pop up.
+
+Email traces:
+- Zip files (Right click > Send to > Compressed folder) and send.
```

Comment:
Should this script auto zip the files?

Reply:
can we add instructions on how to add a provider in this script?

Reply:
Saredd: will consider that as a improvement.
Sudhaushu: I don't thing we have much request from partner to add provider beside the one we have here so far.  Will consider if there are such request.

Reply:
360 camera DMFT partners keep asking how they can log from their DMFT.